docker/ur-hande-control/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM osrf/ros:humble-desktop-full
```

Comment:
Don't make a new docker image, modify an existing one with the 2 repos, and then adjust the command by adding a `docker/launch-hande-driver.sh` that calls `Podman run ....`

Reply:
A new docker image is added that contains necessary libraries and code to execute ur-driver, ur-moveit, and ur-hande-control docker containers. Readme.md is added with the relevant run commands.
https://github.com/maffettone/erobs/tree/9f0d358470cda04c943f454cd38ed3fd825efaa5/docker/erobs-common-img

Reply:
Does that mean this image can go?